### PR TITLE
Feature/linting rebrushed

### PR DESCRIPTION
**Pull request by schmeter**
I reverted the last approach, try to use eslint only as a first step, then apply prettier in a further step (not within this PR)

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -225,11 +213,11 @@ module.exports = function (grunt) {
             },
             js: {
                 files: ['<%= srcFolder %>/js/**/*.js'],
-                tasks: doLinting ? ['eslint'] : []
+                tasks: []
```
Comment (herschel666) [inline Gruntfile.js]: do you still need to watch those files if there aren't related tasks to run?

File: .sass-lint.yml [hunk 2]
```diff
@@ -1,7 +1,8 @@
 files:
+  include: '**/*.scss'
   ignore:
-    - 'node_modules/**/*.*'
-    - 'src/sass/lib/**/*.*'
+    - 'node_modules/**'
```
Comment (herschel666) [inline .sass-lint.yml]: won't `'node_modules'` and `'src/sass/lib'` also do the trick?

Comment (schmeter) [reply .sass-lint.yml]: nope :(

File: Gruntfile.js (hunk 1 above)
Comment (schmeter) [reply Gruntfile.js]: true, removed

**Pull request merged**